crates/sui-core/src/authority_server.rs

```diff
@@ -413,6 +434,21 @@ impl ValidatorService {
             // to save more CPU.
             return Err(error.into());
         }
+
+        if let Some(validator_tx_finalizer) = validator_tx_finalizer {
+            if let TransactionStatus::Signed(sig) = &info.status {
```

Comment:
Is there a plan to only track transactions in validator_tx_finalizer for `interesting` txns only? Seems we already have such information at signing?